docs/guides/vscode.md

```diff
@@ -155,11 +155,31 @@ The VSCode extension is based on a [language server](https://en.wikipedia.org/wi
 
 If you have environment variables that are needed by the context and the language server, you can use one of these approaches to pass variables to the language server:
 
-- Open VSCode from a terminal that has the variables set
-- Use environment variables pulled from somewhere else dynamically (e.g. a `.env` file) in your config 
+- Open VSCode from a terminal that has the variables set already. 
+  - If you have `export ENV_VAR=value` in your shell configuration file (e.g. `.zshrc` or `.bashrc`) when initializing the terminal by default, the variables will be picked up by the language server if opened from that terminal.
+- Use environment variables pulled from somewhere else dynamically (e.g. a `.env` file) in your root project directory. This will automatically be picked up by the language server through the python environment that the extension uses.
 - Set the environment variables in the python environment that the extension uses. You can find detailed instructions [here](https://code.visualstudio.com/docs/python/environments#_environment-variables)
 
-### Python environment woes
+You can verify that the environment variables are being passed to the language server by printing them in your terminal. 
+
+1. `Cmd +Shift + P` (`Ctrl + Shift + P` in case of Windows) to start the VSCode command bar
+   ![print_env_vars](./vscode/print_env_vars.png)
+2. Select the option: `SQLMesh: Print Environment Variables`
+3. You should see the environment variables printed in the terminal
+   ![terminal_env_vars](./vscode/terminal_env_vars.png)
+
+If you change your environment variables in the terminal during development (e.g. `export DATABASE_PASSWORD='XYZ'`), you must restart the language server for the changes to take effect. You can do this by running the following command in the terminal:
```

Comment:
nit: This could be read as you could run that export in a terminal and then they would appear.

Reply:
```suggestion
If you change your setup during development (e.g., add variables to your shell config), you must restart the language server for the changes to take effect. You can do this by running the following command in the terminal:
```